charts/jaeger/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.53.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
-version: 3.2.0
+version: 3.2.1
```

Comment:
The latest version on main is already `3.3.0` so you will need to rebase and bump it again.